unified-runtime/source/loader/layers/validation/ur_valddi.cpp

```diff
@@ -3864,9 +3864,6 @@ __urdlllocal ur_result_t UR_APICALL urKernelGetSuggestedLocalWorkSize(
   }
 
   if (getContext()->enableParameterValidation) {
-    if (NULL == pGlobalWorkOffset)
```

Comment:
I think pGlobalWorkOffset can be null, at least that's what the OpenCL specification says: https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#clGetKernelSuggestedLocalWorkSizeKHR